simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/StationSimulatorFunctionalTest.java

```diff
@@ -637,6 +636,43 @@ void shouldGetHeartbeatIntervalWithGetVariablesRequest() {
         });
     }
 
+    @Test
+    void shouldSendNotifyReportOnGetBaseReportRequest() {
```

Comment:
I think its not a funcitonal test its an integration test. And I created an API for that with verification feature. PLease take a look at ChangeEvseAvailabilityIntegrationTest

Reply:
Also do we verify that we send GetBaseReportResponse?

Reply:
Added a verification for call result.